Stylesheet/css/variables.css

```diff
@@ -1,11 +1,13 @@
 :root {
-    --st-primary-color: #800000;
-    --st-primary-hover: #CC0000;
-    --st-primary-disabled: #FF5C5C;
+    --st-primary-color: #014465;
```

Comment:
Suggestion: To use hsl format for colors instead of hex codes.

Reply:
Okay I will look into it and change, Thank you